### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by tarunsankhla**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -271,3 +271,16 @@ span {
 .btn:hover {
 	transform: scale(1.05);
 }
+
+
+.copied-clipboard {
+    position: absolute;
+    bottom: 1em;
+    right: 2em;
+    font-size: 0.9rem;
```
Comment (manojsarna) [inline src/App.css]: Suggestion : if computed value is less than 10 px , then use px

**Review by manojsarna — APPROVED** [review]
UI Review : UI can be improved for the social media app

File: src/components/UI/Buttons/buttons.js [hunk 2]
```diff
@@ -1,7 +1,10 @@
 import "./buttons.css";
 
 function NormalButton(props) { 
-    return <button className="normal-btn" style={{backgroundColor: `${props.color}` && "red"}} >{ props.name}</button>
+    return <button className={`normal-btn ${props.class}`}
+        style={{ backgroundColor: `${props.color}` || "red", padding: `${props.padding}` || "3px" }}
+        onClick={props.click}>
+        {props.name}{props.icon}</button>
```
Comment (Debarshi95) [inline src/components/UI/Buttons/buttons.js:7]: Suggestion: You can destructuee the individual properties from props object

File: src/pages/ProfilePage/ProfilePage.jsx [hunk 3]
```diff
@@ -1,10 +1,217 @@
...
+                console.log(response.data(), response.id, setUserData, userID);
+                console.log(response.data()[userID]);
+                setUserData(response.data()[userID]);
+                // setData(res1.data() ?? {});
+
+
+                // get All data
+                console.log(getAllPost());
+                let data = await getAllPost();
+                // if (data.isArray()) {
```
Comment (Debarshi95) [inline src/pages/ProfilePage/ProfilePage.jsx:57]: If value is not re initialized, prefer using const

**Review by Debarshi95 — APPROVED** [review]
Good work. Couldn't found any major issue with your code. Added some suggestions, may be work on them

File: src/App.css (hunk 1 above)
Comment (tarunsankhla) [reply src/App.css]: done

File: src/components/UI/Buttons/buttons.js (hunk 2 above)
Comment (tarunsankhla) [reply src/components/UI/Buttons/buttons.js:7]: ok

File: src/pages/ProfilePage/ProfilePage.jsx (hunk 3 above)
Comment (tarunsankhla) [reply src/pages/ProfilePage/ProfilePage.jsx:57]: thanks done

**Pull request merged**